telegram/error.py

```diff
@@ -149,3 +149,16 @@ class Conflict(TelegramError):
 
     def __reduce__(self) -> Tuple[type, Tuple[str]]:
         return self.__class__, (self.message,)
+
+
+class TelegramDecryptionError(TelegramError):
```

Comment:
While at it, We might want to rename the error - none of the others start with `Telegram`. Suggestions: Just `DecryptionError` or `PassportDecryptionError`

Reply:
PassportDecryptionError sounds more PTB specific, so I'll change it to that.